headers/functions/arm9/itcm.h

```diff
@@ -1,6 +1,7 @@
 #ifndef HEADERS_FUNCTIONS_ARM9_ITCM_H_
 #define HEADERS_FUNCTIONS_ARM9_ITCM_H_
 
+void CopyAndInterleave(int* dst, int* src, int len, int val);
```

Comment:
Are these really 4-byte signed `int` values? The description makes me think `dst` should be `uint32_t*`, `src` should be `uint16_t*`, and `val` should be `uint8_t`, though I haven't looked at what the access pattern looks like.

Reply:
The accesses are all ldrh/strh so I suppose both should be uint16_t*